src/overlays/ovl_i4/fox_sz.c

```diff
@@ -1,55 +1,1853 @@
+                    func_800182F4(0x103200FF);
+                    func_800182F4(0x113200FF);
+                }
+
+                gCsFrameCount = 1280;
+
+                player->unk_1D0 = 10;
+                player->timer_1F8 = 1000;
+
+                func_8001CA24(0U);
+                Audio_KillSfx(player->sfxPos);
+                func_800A6148();
+
+                player->pos.x = 0.0f;
+                player->pos.y = 0.0f;
+                player->pos.z = 0.0f;
+
+                player->unk_0E4 = 0.0f;
+                player->unk_234 = 1;
+                player->unk_0E8 = 180.0f;
+                player->unk_0EC = 0.0f;
+                player->unk_114 = 0.0f;
+                player->unk_4D8 = 0.0f;
+                player->unk_0D0 = 40.0f;
+
+                D_80177978 = 0.0f - (200.0f * var_fv1);
+                D_80177980 = player->pos.y;
+                D_80177988 = player->pos.z + 12000.0f;
+                D_801779A0 = player->pos.x;
+                D_801779B8 = player->pos.y;
+                D_801779C0 = player->pos.z;
+
+                if (gTeamShields[1] > 0) {
+                    func_i4_8019C574(actor1, 0);
+                }
+                if (gTeamShields[2] > 0) {
+                    func_i4_8019C574(actor3, 1);
+                }
+                if (gTeamShields[3] > 0) {
+                    func_i4_8019C574(actor2, 2);
+                }
+                func_i4_8019C574(actor0, 3);
+                D_80178340 = D_80178358 = 255;
+            }
+            break;
+
+        case 10:
+            D_80178340 = D_80178358 = 0;
+            D_801779A0 = player->pos.x;
+            D_801779B8 = player->pos.y;
+            D_801779C0 = player->pos.z;
+
+            if (gCsFrameCount == 1620) {
+                player->unk_1D0 += 1;
+                D_80177A48[0] = 0.0f;
+                D_80177A48[1] = 0.0f;
+                D_80177A48[2] = 0.0f;
+                D_80177A48[3] = 300.0f;
+
+                for (i = 0; i < 200; i++) {
+                    gObjects58[i].obj.status = 0;
+                }
+            }
+            break;
+
+        case 11:
+            Matrix_RotateY(gCalcMatrix, D_80177A48[1] * M_DTOR, 0);
+            D_80177A48[1] += D_80177A48[2] * var_fv1;
+
+            src.x = -800.0f * var_fv1;
+            src.y = D_80177A48[3];
+            src.z = 0.0f;
+
+            Matrix_MultVec3fNoTranslate(gCalcMatrix, &src, &dest);
+
+            Math_SmoothStepToF(&player->unk_0D0, 0, 0.05f, 1.0f, 0);
+            Math_SmoothStepToF(&actor1->fwork[0], 0, 0.05f, 1.0f, 0);
+            Math_SmoothStepToF(&actor3->fwork[0], 0, 0.05f, 1.0f, 0);
+            Math_SmoothStepToF(&actor2->fwork[0], 0.0f, 0.05f, 1.0f, 0);
+            Math_SmoothStepToF(&actor0->fwork[0], 0.0f, 0.05f, 1.0f, 0);
+            Math_SmoothStepToF(D_80177A48, 0.02f, 1.0f, 0.001f, 0);
+
+            if (gCsFrameCount > 2000) {
+                Math_SmoothStepToF(D_80177A48 + 3, 50.0f, 0.05f, 1.5f, 0);
```

Comment:
```suggestion
                Math_SmoothStepToF(&D_80177A48[3], 50.0f, 0.05f, 1.5f, 0);
```
Same for the others.